sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -228,7 +228,7 @@ export const SDK_VERSION: string;
 export interface SetRoleDefinitionOptions extends coreHttp.OperationOptions {
     assignableScopes?: KeyVaultRoleScope[];
     description?: string;
-    permissions?: KeyVaultPermission[];
+    permissions?: Partial<KeyVaultPermission>[];
```

Comment:
What do we think about this? Should we instead have

```ts
export type KeyVaultPermissionToSet = Partial<KeyVaultPermission>
```

I couldn't come up with a great name, so I just inlined this. Any suggestions?